src/main/kotlin/uk/co/craigbass/pratura/boundary/checkout/AddShippingAddress.kt

```diff
@@ -5,13 +5,13 @@ import uk.co.craigbass.pratura.boundary.checkout.AddShippingAddress.Request
 
 interface AddShippingAddress : SynchronousUseCase<Request, Unit> {
   data class Request(
-    val name: String,
+    val name: String?,
     val companyName: String?,
-    val addressLine1: String,
+    val addressLine1: String?,
     val addressLine2: String?,
     val addressLine3: String?,
-    val city: String,
-    val province: String,
-    val zipcode: String
+    val city: String?,
+    val province: String?,
+    val zipcode: String?
```

Comment:
It seems odd to name this field after the US-specific ZIP Code when in the majority of cases that's not what it will contain (including all your tests)

Reply:
Yeah potentially. My thought was zipcode was more ubiquitous, but perhaps that is not the case